.gitignore

```diff
@@ -58,3 +58,7 @@ gradle-app.setting
 proguard-mapping.txt
 proguard-seeds.txt
 proguard-usage.txt
+
+*.asc
+*.md5
+*.sha1
```

Comment:
Git is complaining about no newline at end of file.  Not a deal breaker, just Git being Git.